### PR TITLE
Newsletter-134:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

**Comment by bitschmidty** [comment]
@AkioNak if you have time to review, let me know. Otherwise we can move forward without the content review.

**Comment by AkioNak** [comment]
@bitschmidty Sure, I will review.

File: _posts/ja/newsletters/2021-02-03-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: 'Bitcoin Optech Newsletter #134'
+permalink: /ja/newsletters/2021/02/03/
+name: 2021-02-03-newsletter-ja
+slug: 2021-02-03-newsletter-ja
+type: newsletter
+layout: newsletter
+lang: ja
+---
+今週のニュースレターは、Taproot有効化後のスクリプト言語の小さな変更がコントラクトの柔軟性を高める方法に関するブログ投稿のリンクと、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-03-newsletter.md:10]:
```suggestion
今週のニュースレターは、Taproot有効化後のScript言語に小さな変更をすることでコントラクトの柔軟性を高める方法に関するブログ投稿のリンクと、
```

For wording and readability.

File: _posts/ja/newsletters/2021-02-03-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+人気のBitcoinインフラストラクチャソフトウェアの注目すべき変更点を含む通常のセクションを掲載しています。
+
+## ニュース
+
+- **BIP340と`OP_CAT`による`OP_CHECKSIGFROMSTACK`の複製:**
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-03-newsletter.md:15]:
```suggestion
- **BIP340と`OP_CAT`による`OP_CHECKSIGFROMSTACK`の模倣:**
```

File: _posts/ja/newsletters/2021-02-03-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  Andrew Poelstraは、[ElementsProject.org][]の[OP_CHECKSIGFROMSTACK][topic op_checksigfromstack] (`OP_CSFS`) opcode
+  の機能を、現在提案されている[BIP340][]の[Schnorr署名][topic schnorr signatures]の仕様と2010年半ばまでBitcoinの一部であった
+  （そして再導入について言及されることの多い）[OP_CAT][csfs cat] opcodeを使ってBitcoinに実装することについて[ブログ記事][poelstra 340cat]を執筆しました。
+  BitcoinでCSFSのような動作を有効にすることで、トランザクションに事前に署名することなく[Covenants][topic covenants]やその他の高度なコントラクトを作成することが可能になり、
+  複雑さや保存する必要のあるデータ量を削減することができる可能性があります。この記事の最後には、シリーズの後半の記事へのお誘いがります(リンクは私達が追加):
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-03-newsletter.md:20]:
```suggestion
  複雑さや保存する必要のあるデータ量を削減することができる可能性があります。この記事の最後には、シリーズの後半の記事へのお誘いがあります(リンクは私達が追加):
```

typo

File: _posts/ja/newsletters/2021-02-03-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  > [Vault][topic vaults]を構築する方法についてお話します。最後の記事では、[Miniscript][topic miniscript]
+  > のアドホックな拡張と、そのためのソフトウェアをメンテナンス可能な方法で開発する方法についてお話します。"
+
+## 注目すべきコードとドキュメントの変更
+
+*今週の[Bitcoin Core][bitcoin core repo]、[C-Lightning][c-lightning repo]、[Eclair][eclair repo]、[LND][lnd repo]、[Rust-Lightning][rust-lightning repo]、
+[libsecp256k1][libsecp256k1 repo]、[Hardware Wallet Interface (HWI)][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、[BTCPay Server][btcpay server repo]、
+[Bitcoin Improvement Proposals（BIP）][bips repo]、および[Lightning BOLTs][bolts repo]の注目すべき変更点。*
+
+- [Bitcoin Core #20226][]では、ウォレットのための新しい`listdescriptors` RPCメソッドが追加されました。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-03-newsletter.md:33]:
```suggestion
- [Bitcoin Core #20226][]では、ウォレットのための新しいRPCメソッドとして `listdescriptors` が追加されました。
```

File: _posts/ja/newsletters/2021-02-03-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  最近の[0.21.0 ソフトウェアリリース][news132 bitcoin core v0.21]に含まれる[PR #16528][news96 descriptor wallets]では、
+  [Descriptor][topic descriptors]ウォレットのサポートが追加されいました。この新しいRPCメソッドは、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-03-newsletter.md:35]:
```suggestion
  [Descriptor][topic descriptors]ウォレットのサポートが追加されました。この新しいRPCメソッドは、
```
typo

File: _posts/ja/newsletters/2021-02-03-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  DescriptorウォレットにインポートされたすべてのDescriptorをリストアップします。
+
+- [Bitcoin Core GUI #163][]は、GUIピア詳細領域の*Direction*フィールドをDirectionとピア接続のタイプの両方を表示する*Connection Type*に置き換えます。
+  詳細については、Connection Typeフィールド名の上にカーソルを置くと以下のツールチップが表示されます。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-03-newsletter.md:39]:
```suggestion
  詳細については、Connection Typeフィールド名の上にカーソルを置くと以下のようなツールチップが表示されます。
```
clarifying

File: _posts/ja/newsletters/2021-02-03-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+    {:.center}
+    ![Illustration of GUI peer detail connection type](/img/posts/2021-02-gui-peer-connection-type.png)
+
+- [HWI #430][]では、`displayaddress`コマンドでTrezor Oneのマルチシグアドレス用の[BIP32][] 拡張公開鍵（xpub）を表示できるようになりました。
+
+- [HWI #415][]は、`getkeypool`と`displayaddress`コマンドを更新し、`--sh_wpkh`と`--wpkh`オプションを`--addr-type`オプションに置き換えた結果、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-03-newsletter.md:46]:
```suggestion
- [HWI #415][]は、`getkeypool`と`displayaddress`コマンドを更新し、`--sh_wpkh`と`--wpkh`オプションを`--addr-type`オプションに置き換え、
```

This is not result, but intentional change. So '結果' is not necessary here.

**Review by AkioNak — COMMENTED** [review]
I pointed out a few words to change for Japanese reader to make this newsletter more readable.

**Comment by azuchi** [comment]
Thanks @AkioNak! I fixed.

**Comment by AkioNak** [comment]
@azuchi @bitschmidty It looks good to me.

**Comment by bitschmidty** [comment]
Merged! Thank you once again @azuchi and @AkioNak !

**Pull request merged**